kcron-core/src/commonMain/kotlin/com/ucasoft/kcron/core/parsers/Parser.kt

```diff
@@ -85,7 +85,13 @@ class Parser {
                 for (dependency in rule.dependencies!!) {
```

Comment:
A suggest to not use non-null assertion. It could be replaced at least with null checking and a warning in the unexpected case.

Reply:
Fixed